src/styles/components/content.scss

```diff
@@ -0,0 +1,27 @@
+.main {
```

Comment:
It's not clear, why the block name is `main` and the file name is `content`
File name should be equal to the block name